screenings.go

```diff
@@ -0,0 +1,142 @@
+package checkr
+
+import (
+	"log"
+	"sync"
+)
+
+// Screenings ...
+type Screenings struct {
+	SSNTrace                     *SSNTrace
+	SexOffenderSearch            *SexOffenderSearch
+	GlobalWatchListSearch        *GlobalWatchListSearch
+	NationalCriminalSearch       *NationalCriminalSearch
+	FederalCriminalSearch        *FederalCriminalSearch
+	CountryCriminalSearches      []*CountryCriminalSearch
```

Comment:
And you'll need to add JSON tags to these and no need to make a pointer reference to the array elements.

Reply:
will do that